Rename typesafeRelease Resolver

eatkins opened the pull request:
This Resolver had the same name as the typesafe ivy resolver specified
in the launcher boot.properties. It was creating a number of verbose
warnings about having multiple resolvers with the same name. I noticed
that the ivy pattern is slightly different for the boot resolver with
this name. It didn't seem to be causing any problems to have both
resolvers.

Fixes #4839

eatkins committed:
Rename typesafeRelease Resolver

This Resolver had the same name as the typesafe ivy resolver specified
in the launcher boot.properties. It was creating a number of verbose
warnings about having multiple resolvers with the same name. I noticed
that the ivy pattern is slightly different for the boot resolver with
this name. It didn't seem to be causing any problems to have both
resolvers.

Fixes #4839

lightbend-cla-validator commented:
At least one pull request committer is not linked to a user. See https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user

eatkins commented:
There seems to be an issue with the typesafe-cla-validator. This is two PRs in a row that asked me to sign the CLA.

eed3si9n commented:
@eatkins It might be related to your git commit email (gmail) not matching up with GitHub user email (ethanatkins.com).

eatkins commented:
@eed3si9n Yeah. I think it may have been related to the domain so I changed my gitconfig. It's weird that this started happening now though because I updated my github email months ago.
